compiler/rustc_graphviz/src/lib.rs

```diff
@@ -204,6 +212,10 @@
 //! ```rust
 //! #![feature(rustc_private)]
 //!
+//! #[allow(unused_extern_crates)]
+//! // extern `rustc_graphviz` explicitly to avoid lint error
+//! extern crate rustc_graphviz;
```

Comment:
And here.

Reply:
https://github.com/rust-lang/rust/pull/106621#discussion_r1080574124